src/main/java/task/Main.java

```diff
@@ -0,0 +1,63 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    private static final String TEMPLATE = "Name: %s, salary: %s, balance: %s";
+
+    public static void main(String[] args) {
+        List<Person> people = DataGenerator.generatePeople();
+        List<Person> peopleWithSuspiciousBankAccounts = new ArrayList<>();
+
+        people.stream()
+                .filter(person -> person.getBankAccount().isActive()
+                        && person.getBankAccount().getBalance() > person.getSalary())
+                .filter(person -> getIncomeSinceAccountStartDate(person) > person.getBankAccount().getBalance())
+                .forEach(peopleWithSuspiciousBankAccounts::add);
+
+        for (int i = 0; i < people.size(); i++) {
+            Person person = people.get(i);
+            if (person.getBankAccount().isActive() && person.getBankAccount().getBalance() > person.getSalary()) {
+                if (getIncomeSinceAccountStartDate(person) <= person.getBankAccount().getBalance()) {
```

Comment:
if (getIncomeSinceAccountStartDate(person) <= person.getBankAccount().getBalance())
continue;
why not vice versa?

Reply:
Fixed